tests/bootstrap.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
 define("MOCK_MINIMUM_VERSION", "0.40.0");
+define("MOCK_DOMAIN", getenv("STRIPE_MOCK_DOMAIN") ?: 'localhost');
 define("MOCK_PORT", getenv("STRIPE_MOCK_PORT") ?: 12111);
 
 // Send a request to stripe-mock
-$ch = curl_init("http://localhost:" . MOCK_PORT . "/");
+$mock_url = MOCK_DOMAIN . ":" . MOCK_PORT . "/";
+$ch = curl_init($mock_url);
 curl_setopt($ch, CURLOPT_HEADER, 1);
 curl_setopt($ch, CURLOPT_NOBODY, 1);
 curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
 $resp = curl_exec($ch);
 
 if (curl_errno($ch)) {
-    echo "Couldn't reach stripe-mock at `localhost:" . MOCK_PORT . "`. Is " .
+    echo "Couldn't reach stripe-mock at `" . $mock_url . "`. Is " .
```

Comment:
We probably _don't_ want the protocol displayed here, so could you change this like so:
```suggestion
    echo "Couldn't reach stripe-mock at `" . MOCK_HOST . ":" . MOCK_PORT . "`. Is " .
```